Converters/Tekla/Speckle.Converters.TeklaShared/ToSpeckle/Helpers/ClassPropertyExtractor.cs

```diff
@@ -33,18 +31,10 @@ public ClassPropertyExtractor() { }
     return properties;
   }
 
-  private void AddBeamProperties(TSM.Beam beam, Dictionary<string, object?> properties)
-  {
```

Comment:
I left them separate intentionally to add some different properties in the future.

Reply:
I see - we can speciate them again if necessary later on then